<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSearch</class>
 <widget class="QDialog" name="DialogSearch">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelOffset">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Offset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditOffset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSize">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEditX" name="lineEditSize">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelMinLength">
       <property name="text">
        <string>Min length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMinLength">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelEndianness">
       <property name="text">
        <string>Endianness:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBoxX" name="comboBoxEndianness">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonSearch">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="listWidgetMods">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string notr="true">ANSI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">UNICODE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Crypto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Links</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableViewANSI">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEditX" name="lineEditFilter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>25</height>
            </size>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButtonX" name="pushButtonDump">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Dump</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButtonX" name="pushButtonHEX">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Hex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButtonX" name="pushButtonDumpAll">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Dump all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButtonX" name="pushButtonSaveResult">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Save result</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButtonX" name="pushButtonOK">
           <property name="minimumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>77</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QComboBoxX</class>
   <extends>QComboBox</extends>
   <header>qcomboboxx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
